crates/autopilot/src/lib.rs

```diff
@@ -569,22 +569,22 @@ pub async fn main(args: arguments::Arguments) -> ! {
         args.limit_order_price_factor
             .try_into()
             .expect("limit order price factor can't be converted to BigDecimal"),
-        true,
+        !args.enable_colocation,
```

Comment:
nit: The argument is currently named `store_in_db`. Maybe it makes sense to make it refer to the colocated driver to avoid any confusion.

Reply:
The argument controls whether the solvable_orders module should store the auction in the db, so from that pov it says what it does. Whether it should store in the DB depends on whether colocation is enabled. But the module itself doesn't need to know that. I can see either perspective being reasonable. Will think about it again but not sure what is better.

Reply:
You're right @vkgnosis